src/Aspire.Hosting/Oracle/OracleDatabaseConnectionResource.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// A resource that represents a Oracle Database connection.
+/// </summary>
+/// <param name="name">The name of the resource.</param>
+/// <param name="connectionString">The Oracle Database connection string.</param>
+public class OracleDatabaseConnectionResource(string name, string? connectionString) : Resource(name), IOracleDatabaseResource
```

Comment:
You can remove this type and its associated extension method. We've opted not to have XYZConnection resource types for now.

Reply:
Done